bridges/opentracing/lib/opentelemetry/bridge/opentracing/scope.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # Scope provides a means of referencing an OTelemetry Tracer's Context as
+      # an OTracing scope
+      class Scope
+        def span
+          OpenTelemetry.tracer.current_span
+        end
+
+        def close
+          OpenTelemetry.tracer.current_span.finish
+        end
+      end
```

Comment:
I *think* what we need here is management of a stack of `Scope`s, so:
```ruby
class Scope
  attr_reader :span

  def initialize(manager, span, finish_on_close)
    @manager = manager
    @parent = manager.active
    @span = span
    @finish_on_close = finish_on_close
  end

  def close
    @span.finish if @finish_on_close
    @manager.active = @parent
  end
end
```

Where:
```ruby
class ScopeManager
  KEY = :__opentelemetry_opentracing_scope__
  private_constant :KEY

  def active
    Thread.current[KEY]
  end

  def active=(scope)
    Thread.current[KEY] = scope
  end

  def activate(span, finish_on_close: true)
    self.active = Scope.new(self, span, finish_on_close)
  end
end
```